tools/tests/karma.conf.js

```diff
@@ -41,7 +42,8 @@ module.exports = function (config) {
       '**/rangeslider/*.js',
       '**/automatic-grid/*.js',
       '**/radio-button/*.js',
-      '**/switch/*js'
+      '**/switch/*js',
+      '**/form-control/*js'
```

Comment:
```suggestion
      '**/switch/*.js',
      '**/form-control/*.js'
```

The tests are not executed. 🗡️

Reply:
It makes sense that * should also match a . but the reported didn't have logs about the form control tests.
Will check again, maybe we checked the wrong branch.